test/unit/parser.test.js

```diff
@@ -55,6 +55,39 @@ test('includeFile replaces <include> with <div>', async () => {
   expect(result).toEqual(expected);
 });
 
+test('includeFile replaces <include if-exists> with empty <div> if file does not exist', async () => {
```

Comment:
`'includeFile replaces <include src="doesNotExist.md" if-exists> with empty <div>'` makes the intention clearer.